src/moonlink_connectors/src/pg_replicate/conversions/text.rs

```diff
@@ -351,13 +354,14 @@ impl TextFormatConverter {
                         }
                         Kind::Array(_) => {
                             // TODO: Multi-dimensional arrays not yet implemented
-                            // TODO: Need to print out the unsupported type (included in error type)
-                            Err(FromTextError::InvalidConversion())
+                            Err(FromTextError::InvalidConversion(format!(
+                                "Multi-dimensional arrays"
+                            )))
                         }
-                        _ => Err(FromTextError::InvalidConversion()),
+                        other => Err(FromTextError::InvalidConversion(format!("{:?}", other))),
```

Comment:
The error message here is not clear enough, `other` is an internal type for array, we should include the full type.